CHANGELOG.md

```diff
@@ -1,25 +1,73 @@
 
 ## [2.3.0](https://launchpad.net/mixxx/+milestone/2.3.0) (Unreleased)
-
-* Add support for multi-threaded track analysis. [lp:1641153](https://bugs.launchpad.net/mixxx/+bug/1641153)
-* Add built-in support for MP3 encoding. LAME is now a required dependency. [lp:1294128](https://bugs.launchpad.net/mixxx/+bug/1294128)
-* Add support for searching for empty fields. (e.g. crate:""). [lp:1788086](https://bugs.launchpad.net/mixxx/+bug/1788086)
-* Unify skin controls for better consistency of settings across skins. [lp:1740513](https://bugs.launchpad.net/mixxx/+bug/1740513)
-* Remove VAMP plugin support. vamp-plugin-sdk and vamp-hostsdk are no longer required dependencies.
-* Remove SoundSource plugin support. [lp:1792747](https://bugs.launchpad.net/mixxx/+bug/1792747)
+### Hotcues ###
+* Add hotcue colors and custom labels by right clicking hotcue buttons or right clicking hotcues on overview waveforms [#2016](https://github.com/mixxxdj/mixxx/pull/2016) [#2520](https://github.com/mixxxdj/mixxx/pull/2520) [#2238](https://github.com/mixxxdj/mixxx/pull/2238) [#2560](https://github.com/mixxxdj/mixxx/pull/2560) [#2557](https://github.com/mixxxdj/mixxx/pull/2557) [#2362](https://github.com/mixxxdj/mixxx/pull/2362)
+* Mouse hover cues on overview waveform to show time remaining until the cue [#2238](https://github.com/mixxxdj/mixxx/pull/2238)
+
+### Hotcue & Track Colors ###
+* Add configurable color per track [#2470](https://github.com/mixxxdj/mixxx/pull/2470) [#2539](https://github.com/mixxxdj/mixxx/pull/2539) [#2545](https://github.com/mixxxdj/mixxx/pull/2545) [#2630](https://github.com/mixxxdj/mixxx/pull/2630) [lp:1100882](https://bugs.launchpad.net/mixxx/+bug/1100882)
+* Add customizable color palettes for hotcue and track colors [#2530](https://github.com/mixxxdj/mixxx/pull/2530) [#2589](https://github.com/mixxxdj/mixxx/pull/2589)
+* Add hotcue color find-and-replace tool [#2547](https://github.com/mixxxdj/mixxx/pull/2547)
+
+### Importing From Other DJ Software ###
+* Import cue points, track colors, one loop and playlists from Rekordbox USB drives [#2119](https://github.com/mixxxdj/mixxx/pull/2119) [#2555](https://github.com/mixxxdj/mixxx/pull/2555) [#2543](https://github.com/mixxxdj/mixxx/pull/2543)
+  * Note: The first Rekordbox memory cue is imported for the main cue button in Mixxx and the remaining Rekordbox memory cues are imported as Mixxx hotcues, starting with the next hotcue number after the last hotcue from Rekordbox.
+* Import cue points, track colors, one loop and playlists from Serato file tags & database [#2480](https://github.com/mixxxdj/mixxx/pull/2480) [#2526](https://github.com/mixxxdj/mixxx/pull/2526) [#2499](https://github.com/mixxxdj/mixxx/pull/2499) [#2495](https://github.com/mixxxdj/mixxx/pull/2495)
+
+  * Note: Mixxx does not yet support multiple loops per track. We are working on this for Mixxx 2.4. In Mixxx 2.3, if you import a track with multiple loops from Rekordbox or Serato, only the first loop will be imported.
```

Comment:
Should this hint also go to the Rekordbox root view?
If the feature looks like in 2.2 users familiar with the workflow won't read the changelog, right?

Reply:
I moved it above both the Rekordbox and Serato bullet points. Does that work?

Reply:
?
I was trying to say: those notes should probably also be in the root view of those features in the _Library_.

Reply:
Oh, good idea. Want to make it happen?